kornia/feature/matching.py

```diff
@@ -409,12 +409,23 @@ def matching_function(self, d1: Tensor, d2: Tensor, dm: Optional[Tensor] = None)
         else:
             raise NotImplementedError
 
-    def forward(self, desc1: Tensor, desc2: Tensor, normalize: bool = False) -> Tuple[Tensor, Tensor, Optional[int]]:
+    def forward(
+        self,
+        desc1: Tensor,
+        desc2: Tensor,
+        normalize: bool = False,
+        fast: bool = False,
+        subset_size: int = 1000,
+    ) -> Tuple[Tensor, Tensor, Optional[int]]:
         """
         Args:
             desc1: Batch of descriptors of a shape :math:`(B1, D)`.
             desc2: Batch of descriptors of a shape :math:`(B2, D)`.
             normalize: bool to decide whether to normalize descriptors to unit norm.
+            fast: bool to decide whether to determine optimal number of rotations
+                using only a subset of the descriptions.
+                This is only used if `self.steer_mode` is `global`.
+            subset_size: The subset size to use if `fast` is True.
```

Comment:
instead of using a fast bool flag, to be simpler, what do you think of using if subset_size is None, then fast=False, otherwise use fast mode.

Reply:
Good idea, fixed.